### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by Phill93** [zettajs/zetta#376]
Hi,

zetta seems to be a nice lightweight IoT server. I want to use it.

But i think the project is abandoned, am I right?

**Comment by AdamMagaluk** [comment]
Yes and no. We're still here but with little support. All the maintainers are busy with other jobs.

We're happy to take contributions and help where we can. Zetta upgrades have been a bit blocked on some spdy library issues when upgrading.

**Comment by rambo-droid** [comment]
> Yes and no. We're still here but with little support. All the maintainers are busy with other jobs.
> 
> We're happy to take contributions and help where we can. Zetta upgrades have been a bit blocked on some spdy library issues when upgrading.

Hi Adam, my name is Sanders, and I am a researcher at Politecnico di Milano. We are doing research on Open Source IoT Platforms and I am looking to forward to talk directly with developers/users on the platform. 
Are you available for a little chat?

**Comment by amos-kibet** [comment]
Hello,

I just found out about this IoT platform and was wondering if it is still being maintained or what the current state is. It seems to be a cool product and I am interested in knowing whether it is receiving updates. Would love to contribute as well.

**Comment by kevinswiber** [comment]
@amos-kibet We built this project while we were working at a company named Apigee.  In 2016, Apigee was acquired by Google Cloud Platform.  I don't believe any of us original developers still have access to this repo, as it's now owned by Google.  I'd support a fork if anyone were so inclined.

**Comment by amos-kibet** [comment]
@kevinswiber would you mind if we do a fork? I would be happy to advance this project

**Comment by amos-kibet** [comment]
A fork would be good, and I will be happy to contribute.

**Comment by AdamMagaluk** [comment]
Opened a new PR since i lost access to the repo after leaving Google. https://github.com/zettajs/zetta/pull/379

I also published version zetta@1.6.0 that incorporates all these changes. The latest version of Zetta should work with Node.js 18 and  other newer versions of Node.

**Comment by amos-kibet** [comment]
Checking the PR.

**Comment by nikelborm** [comment]
Hi, guys!
Speaking on the topic of abandonment, I clicked to the link of your site and seems like it was hacked and now showing casinos on the main page

![image](https://github.com/user-attachments/assets/b3c8b927-37fd-4b18-8a5a-61e8dee308ed)
![image](https://github.com/user-attachments/assets/8fcd82bd-57f8-477f-acd2-35672734796f)

**Comment by AdamMagaluk** [comment]
Yeah... unfortunately i got booted from the org after leaving Google and cannot make any changes to the README. I believe all other contributors lost access as well. @kevinswiber Do you still have access?

**Comment by kevinswiber** [comment]
@AdamMagaluk Yes. I added you as an admin, as well. No guarantee on how long that lasts. But maybe we can make some updates in the meantime.
